app-dicts/myspell-lv/Manifest

```diff
@@ -1,2 +1,3 @@
 DIST lv_LV-1.0.0.zip 369295 SHA256 6756180a7cde97b20b440857a9a7b9df411723dfd3165adab40865f82b249957 SHA512 c97509edc3e1c018f6a5237fe56d4fc4da4f18a69b3003284bbff6dbb13f57a575c59d0763266af2d83ec0cf51b0e8ae67b20a383f41251e0002cf01538b823d WHIRLPOOL d40fe66f95d8b906937081c26bf77fb5a683f4630f4c87447647676818ca35a23d29558bd48ad39dcc19a9d2c6a070ccb75bf140f11dbdb184aa8c84d944c7b0
 DIST lv_LV-1.1.0.zip 371240 SHA256 1845ee1ce23614ccb0ad29b1441827a53d137d7afdd7d40a835e996e8f899aa3 SHA512 4083311c1f5afd315ba348f92a221155137f2d32c0cd3f8869ba43cbbdecb1375c8dc444ef2b70631a18265e5ec72517a9d6ca8cad7e5e39a570a98ee075bfe8 WHIRLPOOL ef930bb36d4d343f831e141e52ab837e1ac73dc242aa7bd6c4aac7db5bf8359a239a442a3e410f171c51362a7474f6c0277796168fff74e070121a4b30bce7d4
+DIST lv_LV-1.3.0.zip 374356 SHA256 2f8856f58f2dcc6153593c142cfc7db02697706a1d84c717cacec943fbe40a90 SHA512 fbb53475870cd0200403e75ea3f84c8534b4fac349093515821e98a8e1f3bd839fdec0543a2e4ded1bacab93052babbf8ffbee4742d291bfc91f2d768b4ac445 WHIRLPOOL e715fd7e066f8a7bc100b5554eaaa778139f8683705f72da2a1337abe3006586166de815cb18da5ae3cf2b31987873da6b66a8e33b3f5ca9da8ef93b5b8fcf59
```

Comment:
One more thing though: this is not a proper filename for a distfile — it's making it hard to figure out where it came from, and it's easy for it to collide with another package. I'll add `-> ${P}.zip` to SRC_URI to solve this.